lib/module/esp-wroom-32.yaml

```diff
@@ -0,0 +1,45 @@
+datasheet: https://espressif.com/en/products/hardware/esp-wroom-32/overview
+description: ESP32 (ESP-WROOM-32 module)
+footprint: agg:ESP-WROOM-32
+name: ESP-WROOM-32
+ordercodes: []
+pins:
+- - - [VCC, 2, pwrin]
+    - [GND, 1, pwrin]
+    - [GND, 15, pwrin]
+    - [GND, 38, pwrin]
+    - [GND, 39, pwrin]
+  - - [EN, 3, in]
+    - [NC, 32, nc]
+  - - [TXD-GPIO1, 35, bidi]
+    - [RXD-GPIO3, 34, bidi]
```

Comment:
Do you think these should be TXD0 and RXD0 instead of just TXD and RXD, as per the datasheet?